### DISCUSSION TITLE
How can I break layout in nextjs?

### DISCUSSION THREAD

**Discussion by ShourovRoy** [vercel/next.js#66588] [App Router]
In remix or sveltekit I can break layouts, For example if I have a Blog route which has a layout and the blog has 2 nested routes, 1 nested route will use the parent blog layout but another one can break that layout or totally empty the layout.

Is that possible in nextjs?

**Comment by hyjoong** [comment]
It might be helpful to use Parallel Routes. In your top-level component, you can conditionally render either the children or the Parallel Routes based on the specific route.

https://nextjs.org/docs/app/building-your-application/routing/parallel-routes

**Reply by ShourovRoy** [reply]
Thats not i was looking for.

**Reply by serdar-webdev** [reply]
Hi folk, can you explain more about your case, do you wanna break parent layout dynamic or static? You want to change parent layout from code?
Because i have an idea but it can be hard to implement. ( but it will work )

**Reply by ShourovRoy** [reply]
> Hi folk, can you explain more about your case, do you wanna break parent layout dynamic or static? You want to change parent layout from code? Because i have an idea but it can be hard to implement. ( but it will work )

In Sveltkit kit we can break the parent layout of as nested child using @ suffix and in remix a trailing _ underscore flag do the same thing. Is there anything in nextjs?

**Reply by serdar-webdev** [reply]
https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-breaking-out-of-layouts; next js can do this staff; in next js we also have route grouping; 
here:
https://nextjs.org/docs/app/building-your-application/routing/route-groups#opting-specific-segments-into-a-layout

**Reply by ShourovRoy** [reply]
> https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-breaking-out-of-layouts; next js can do this staff; in next js we also have route grouping; here: https://nextjs.org/docs/app/building-your-application/routing/route-groups#opting-specific-segments-into-a-layout

I have seen this and its not what can break the parent layout as child of that nested route. In sveltkit and remix. The nested child can opt-out its parent layout and use the only root layout. Is that possible?

**Reply by serdar-webdev** [reply]
Unfortunetly no, as i know it's not possible, you have to use route grouping, here you can create different layouts for each page or you can write some logic inside layout file, to specify what should render.
By the way, i cannot imagine in which cases you need that, can you give me examples, im just insteresting
